### PR TITLE
add tool to generate savepoint file

### PR DESCRIPTION
**Pull request by WangXiangUSTC**

### CODE REVIEW

**Comment by WangXiangUSTC** [comment]
@GregoryIan PTAL

File: Makefile [hunk 1]
```diff
@@ -21,6 +21,9 @@ checker:
 dump_region:
 	$(GO) build -ldflags '$(LDFLAGS)' -o bin/dump_region ./dump_region
 
+gen_meta:
```
Comment (IANTHEREAL) [inline Makefile]: generate_binlog_position

File: gen_meta/README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+## gen_meta
+
+gen_meta is a tool to generate fake drainer savepoint file.
+
+## How to use
+
+```
+Usage of gen_meta:
+  --config string
```
Comment (IANTHEREAL) [inline gen_meta/README.md]: use cmd arguments, it's not worth for config file

File: gen_meta/config.go [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
```
Comment (IANTHEREAL) [inline gen_meta/config.go]: too complicate

**Comment by IANTHEREAL** [comment]
LGTM

**Pull request merged**